skore/src/skore/sklearn/_plot/metrics/precision_recall_curve.py

```diff
@@ -885,3 +885,39 @@ def _compute_data_for_display(
             ml_task=ml_task,
             report_type=report_type,
         )
+
+    def frame(self) -> DataFrame:
+        """Get the data used to create the precision-recall curve plot.
+
+        Returns
+        -------
+        DataFrame
+            A DataFrame containing the precision-recall curve data with columns:
+            - estimator_name: Name of the estimator
+            - split_index: Cross-validation fold ID (if applicable)
```

Comment:
```suggestion
            - split_index: Cross-validation fold ID (may be null)
```
"if applicable" sounds like the column might not be there, but it always is

Reply:
Done